kratos/utilities/geometry_utilities.h

```diff
@@ -46,7 +46,7 @@ class GeometryUtils
      * @param the volume of the element
      */
     static inline void CalculateGeometryData(
-        Element::GeometryType& geom,
```

Comment:
this should be a separate PR!

Reply:
It is. You can actually find it here: https://github.com/KratosMultiphysics/Kratos/pull/4032

The reason why you see these commit in this branch also is just because I merged the branch https://github.com/KratosMultiphysics/Kratos/tree/core/fix-geometry-utilities into this one... did I understand you wrong?